internal/pkg/cli/env_package.go

```diff
@@ -131,6 +134,14 @@ func newPackageEnvOpts(vars packageEnvVars) (*packageEnvOpts, error) {
 
 // Validate returns an error for any invalid optional flags.
 func (o *packageEnvOpts) Validate() error {
+	if o.showDiff {
+		if o.outputDir != "" {
+			return fmt.Errorf("`--%s` cannot be specified together with `--%s`", diffFlag, stackOutputDirFlag)
+		}
```

Comment:
can we use `cmd.MarkFlagsMutuallyExclusive` instead to get rid of the functionality? (samesies in other commands)